### PR TITLE
feat: SS-96 Initial draft of system boundaries

### PR DESCRIPTION
**Pull request by N4T10N01**
# GitHub Ticket / Description

[SS-96](https://github.com/lilweege/SyntaxSentinels/issues/96)

## Type && Checklist

- [x] **New feature (new APIs, new functional methods)**
  - [ ] Created (or updated) unit test cases (if required)
  - [ ] Passed unit testing for **ALL** cases
  - [ ] Created (or updated) E2E cases (if required)
  - [ ] Passed E2E testing for **ALL** cases (local or/and sandbox)
- [ ] **Refactoring / Enhancement**
  - [ ] This is breaking change
  - [ ] Created (or updated) unit test cases (if required)
  - [ ] Passed unit testing for **ALL** cases
  - [ ] Created (or updated) E2E cases (if required)
  - [ ] Passed E2E testing for **ALL** cases (local or/and sandbox)
- [ ] **Bug-fix**
  - [ ] Type:
    - [ ] Code Issue:
    - [ ] Business Logic Issue:
  - [ ] Created (or updated) unit test cases (if required)
  - [ ] Passed unit testing for **ALL** cases
  - [ ] Created (or updated) E2E cases (if required)
  - [ ] Passed E2E testing for **ALL** cases (local or/and sandbox)

## Dependency

[Are there any required updates which cannot be done in this repo, or must be done after completing this PR?]

## Choose proper operation and replace the \<placeholder\>: `+semver:<placeholder>`

`breaking` `major` `feature` `minor` `fix` `patch`

+semver:patch

### CODE REVIEW

**Comment by lilweege** [comment]
If you used draw.io or another vector graphics program, please upload the source file so it can be edited.

**Review by N4T10N01 — COMMENTED** [review]
Going to add some definitions and assumptions here for specifying what we are owning as our system. Will align this with other sections too.

File: docs/HazardAnalysis/HazardAnalysis.tex [hunk 1]
```diff
@@ -71,11 +71,9 @@ \section{Scope and Purpose of Hazard Analysis}
...
 \section{System Boundaries and Components}
 
-\wss{Dividing the system into components will help you brainstorm the hazards.
-You shouldn't do a full design of the components, just get a feel for the major
-ones.  For projects that involve hardware, the components will typically include
-each individual piece of hardware.  If your software will have a database, or an
-important library, these are also potential components.}
+\hspace*{-3cm}\includegraphics[height=.58\textheight]{./assets/Component_Border.png}
+
+Diagram shows referential relationships, not workflows. System Boundary above surrounds all components our team will maintain and accept responsibility. A machine above is a server suitable to host code specified. Any machine mentioned above will not be provided to users from our project, they are expected to be self-hosted through technology such as Amazon Web Services (AWS) or Microsoft Azure.
```
Comment (lilweege) [inline docs/HazardAnalysis/HazardAnalysis.tex:76]:
```suggestion
Diagram shows referential relationships, not workflows. System Boundary above surrounds all components our team will maintain and accept responsibility. A machine above is a server suitable to host code specified. Any machine mentioned above will not be provided to users from our project, they are expected to be self-hosted either on the user's own servers, or through technology such as Amazon Web Services (AWS) or Microsoft Azure.
```

File: docs/SRS-Volere/SRS.tex [hunk 2]
```diff
@@ -1440,9 +1440,9 @@ \section*{Appendix --- Reflection}
 
         \item \textbf{Julian Cecchini:} 
         \begin{itemize}
-            \item \textbf{Server Deployment}: Julian will pursue online articles to learn this. He finds maintaining attention for long YouTube videos can be cumbersome and if he's interrupted, it's easier to pick up off online articles than YouTube videos. Also, he doesn't feel he has time to trial server deployment in a side project. He lacks experience in server deployment compared to the rest of the team, so he seeks to close the knowledge gap to be able to contribute and gain experience in this area.
+            \item \textbf{Server Deployment}: Julian will pursue online articles to learn this. He finds maintaining attention for long YouTube videos can be cumbersome and if he's interrupted, it's easier to find where he left off in online articles compared to YouTube videos. Also, he doesn't feel he has time to trial server deployment in as a side project. He lacks experience in server deployment compared to the rest of the team, so he seeks to close the knowledge gap to be able to contribute and gain experience in this area.
```
Comment (lilweege) [inline docs/SRS-Volere/SRS.tex:1443]: Please also rebuild the pdf for this file

**Review by lilweege — CHANGES_REQUESTED** [review]

**Comment by lilweege** [comment]
Also make sure you merge the latest HazardAnalysis.pdf

**Review by lucasichen — APPROVED** [review]

File: docs/SRS-Volere/SRS.tex (hunk 2 above)
Comment (lilweege) [reply docs/SRS-Volere/SRS.tex:1441]: There is still no pdf for this

**Review by lilweege — APPROVED** [review]
lgtm

**Pull request merged**